rackcity/utils/change_planner_utils.py

```diff
@@ -362,12 +362,22 @@ def get_cp_modification_conflicts(asset_cp):
     )
     conflicting_asset_message_2 = " now conflicts with a live asset. "
     if asset_cp.is_conflict and asset_cp.related_asset is None:
-        conflicts.append(
-            {
-                "conflict_message": "This asset has been deleted live. "
-                + nonresolvable_message
-            }
-        )
+        if asset_cp.related_decommissioned_asset:
```

Comment:
if a asset cp was decommissioned live it used to think it was deleted AND decommissioned. fixed